### PR TITLE
React Conf Recap Blog Post

### PR DESCRIPTION
**Pull request by jtannady**
React Conf 2021 Recap blog post. Co-authored with @rickhanlonii. 

Open for review! ✨

CC: @acdlite @gaearon

### CODE REVIEW

File: content/blog/2021-12-15-react-conf-2021-recap.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+In previous years, we've used the React Conf stage to deliver industry changing announcements such as [React Native](https://engineering.fb.com/2015/03/26/android/react-native-bringing-modern-web-techniques-to-mobile/) and [React Hooks](https://reactjs.org/docs/hooks-intro.html). This year, we shared our multi-platform vision for React, starting with the release of **React 18** and the **concurrent rendering engine**.
+
+After [piloting React 18](https://engineering.fb.com/2020/05/08/web/facebook-redesign/) with select teams at Meta, and are excited to announce that we're ready to share React 18 with a wider audience. We are releasing the React 18 RC to open source so that members of the React community can begin using the new concurrent rendering engine today.
+
+This release is a milestone of years of collaborative research between the React, React Native, and React Data teams. With this release we're rolling out features that power the [facebook.com](https://www.facebook.com/) site and set the foundation for upcoming features such as **Server Components** and **React Native pre-rendering**.
```
Comment (gaearon) [inline content/blog/2021-12-15-react-conf-2021-recap.md:20]: If I scan the bolded parts then it’s easy to assume we’re saying that Server Components are going to be in 18. Also I expect that the vast majority of our open source users don’t have context on what RN pre-rendering is so we probably shouldn’t emphasise it here.

File: content/blog/2021-12-15-react-conf-2021-recap.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+We also shared how we're raising the bar across many platforms with our [many platform vision](https://reactnative.dev/blog/2021/08/26/many-platform-vision). The foundation set by React 18 will power our investment into platforms such as Desktop and VR.
+
+![A diagram showing the proposed platforms React plans to support](../images/blog/react-conf-2021-recap/multi-platform-vision.png)
+
+## Here are some key highlights from the conference {#key-highlights-from-conf}
+
+* **React 18 RC:** We announced React 18, the foundation for the future of React. This release includes the **new concurrent renderer** that powers the [facebook.com](http://facebook.com/) site, **Suspense** , and many upcoming features such as **Server Components** , and **React Native pre-rendering**. Learn more at [Andrew Clark](https://twitter.com/acdlite) and [Juan Tejada's](https://twitter.com/_jstejada) talk in the [React 18 Keynote](https://www.youtube.com/watch?v=FZ0cG47msEk).
```
Comment (gaearon) [inline content/blog/2021-12-15-react-conf-2021-recap.md:28]: Here too it’s easy to misunderstand it as “this release includes […] server components”

**Review by gaearon — CHANGES_REQUESTED** [review]
We chatted with Rick out of band. We think this is a bit too dense in the amount of information and we need to tweak the format a little bit. Rick will follow up on this but for now let’s not merge yet.

**Comment by rickhanlonii** [comment]
Update: We re-wrote this to be more of an overview.

Here's a mostly complete screenshot (the last few thanks were cut off by Chrome):

blog

File: content/blog/2021-12-17-react-conf-2021-recap.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
+---
+title: "React Conf 2021 Recap"
+author: [jtannady, rickhanlonii]
```
Comment (Amir-Alipour) [inline content/blog/2021-12-17-react-conf-2021-recap.md:3]: author: [[jtannady](https://github.com/jtannady), [rickhanlonii](https://github.com/rickhanlonii)]

Comment (harish-sethuraman) [reply content/blog/2021-12-17-react-conf-2021-recap.md:3]: Blog authors are already in the correct format.

Comment (Amir-Alipour) [reply content/blog/2021-12-17-react-conf-2021-recap.md:3]: aw, i don't saw it

**Review by acdlite — APPROVED** [review]

File: content/blog/2021-12-17-react-conf-2021-recap.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+To try the React 18 RC, upgrade your dependencies:
+
+```
+npm install react@rc react-dom@rc
+```
+
+and switch to the new `createRoot` API:
+
+```
+// before
```
Comment (harish-sethuraman) [inline content/blog/2021-12-17-react-conf-2021-recap.md:41]: Shall we change it like the one below to preview it as JS code? 

\`\`\`js
// before
const container = document.getElementById('root');
ReactDOM.render(<App />, container);
// after
const container = document.getElementById('root');
const root = ReactDOM.createRoot(container);
root.render(<App/>);
\`\`\`

```js
// before
const container = document.getElementById('root');
ReactDOM.render(<App />, container);
// after
const container = document.getElementById('root');
const root = ReactDOM.createRoot(container);
root.render(<App/>);
```

Comment (rickhanlonii) [reply content/blog/2021-12-17-react-conf-2021-recap.md:41]: Thanks, done!

File: content/blog/2021-12-17-react-conf-2021-recap.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+```bash
+npm install react@rc react-dom@rc
+```
+
+and switch to the new `createRoot` API:
+
+```js
+// before
+const container = `document.getElementById('root');`
```
Comment (harish-sethuraman) [inline content/blog/2021-12-17-react-conf-2021-recap.md:42]:
```suggestion
const container = document.getElementById('root');
```

Comment (rickhanlonii) [reply content/blog/2021-12-17-react-conf-2021-recap.md]: Thanks!

File: content/blog/2021-12-17-react-conf-2021-recap.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+Last week we hosted our 6th React Conf.  In previous years, we've used the React Conf stage to deliver industry changing announcements such as [_React Native_](https://engineering.fb.com/2015/03/26/android/react-native-bringing-modern-web-techniques-to-mobile/) and [_React Hooks_](https://reactjs.org/docs/hooks-intro.html). This year, we shared our multi-platform vision for React, starting with the release of React 18 and gradual adoption of concurrent features.
+
+This was the first time React Conf was hosted online, and it was streamed for free, translated to 8 different languages. Participants from all over the world joined our conference Discord and the replay event for accessibility in all timezones. Over 50,000 people registered, with over 60,000 views of 19 talks, and 5,000 participants in Discord across both events.
+
+All the talks are [available to stream online](https://www.youtube.com/watch?v=FZ0cG47msEk&list=PLNG_1j3cPCaZZ7etkzWA7JfdmKWT0pMsa).
+
+Here’s a summary of what was shared on stage:
+
+## React 18 and concurrent features
```
Comment (harish-sethuraman) [inline content/blog/2021-12-17-react-conf-2021-recap.md:14]: can we once run `yarn generate-ids` to generate heading ids?

**Comment by harish-sethuraman** [comment]
@rickhanlonii  Did we actually use the image that has been added inside the markdown? 😅  I don't think we used it.

**Comment by gaearon** [comment]
no, can be deleted

**Pull request merged**